com.unity.multiplayer.mlapi/Runtime/Core/MessageBatcher.cs

```diff
@@ -187,6 +187,7 @@ public void SendItems(int threshold, SendCallbackType sendCallback)
         /// <param name="receiveTime"> the packet receive time to pass back to callback</param>
         public int ReceiveItems(in BitStream messageStream, ReceiveCallbackType receiveCallback, MLAPI.RPCQueueManager.QueueItemType messageType, ulong clientId, float receiveTime)
         {
+            PooledBitStream copy = PooledBitStream.Get();
```

Comment:
use `using (...) { ... }` block instead?

Reply:
Can you elaborate on the _why_, please? I'm more experienced with other languages than C#, so I need to know what effect of `using` would make the code better here. Naively, I'd expect using to simply reduce the scope. Since `copy` would end up in the exact same scope, with `using`, I don't see the advantage.

If the reason is that `using` would somehow affect the disposal of the object, can you elaborate how? Is this an intended feature of the language, or merely a side-effect?

Thanks (and sorry for the many questions)

Reply:
Replying to myself... :-)

Got it. I'll go with `using`, after reading up on it.